ReactViewControl/ReactView.cs

```diff
@@ -86,6 +86,10 @@ public abstract class ReactView : UserControl, IDisposable {
         /// </summary>
         protected virtual ReactViewFactory Factory => new ReactViewFactory();
 
+        protected void RefreshDefaultStyleSheet() {    
```

Comment:
Refresh or Reset? where is this used? by derived classes?

Reply:
I believe it's a refresh since it's the Factory that decides what stylesheet will be used. 
For me a reset would set the DefaultStyleSheet with the initial stylesheet, and wouldn't request the new stylesheet to the factory.

This will be used by the derived classes, such as the `ExtendedReactView`. This one will be listening to environment changes and call the refresh when need. Do you think it should be a `public` in order to have more flexibility to other future use cases. For now, I don't find it necessary.

Reply:
thats ok